configure.py

```diff
@@ -1779,16 +1789,16 @@ def configure_abseil(build_dir, mode):
         build dist-tools-deb: build-submodule-deb tools/java/build/scylla-tools-package.tar.gz
           dir = tools/java
           artifact = $builddir/scylla-tools-package.tar.gz
-        build dist-tools-tar: phony {' '.join(['$builddir/{mode}/dist/tar/scylla-tools-package.tar.gz'.format(mode=mode) for mode in build_modes])}
+        build dist-tools-tar: phony $builddir/{mode}/dist/tar/scylla-tools-package.tar.gz
         build dist-tools: phony dist-tools-tar dist-tools-rpm dist-tools-deb
 
         rule compat-python3-reloc
-          command = mkdir -p $builddir/release && ln -f $dir/$artifact $builddir/release/
+          command = mkdir -p $builddir/release && ln -f $dir/$artifact $builddir/{mode}/
         rule compat-python3-rpm
           command = cd $dir && ./reloc/build_rpm.sh --reloc-pkg $artifact --builddir ../../build/redhat
         rule compat-python3-deb
           command = cd $dir && ./reloc/build_deb.sh --reloc-pkg $artifact --builddir ../../build/debian
-        build $builddir/release/scylla-python3-package.tar.gz: compat-python3-reloc tools/python3/build/scylla-python3-package.tar.gz
+        build $builddir/{mode}/scylla-python3-package.tar.gz: compat-python3-reloc tools/python3/build/scylla-python3-package.tar.gz
```

Comment:
Why is this needed? We don't want to _build_ `scylla-python3-package.tar.gz` multiple times. Instead, we want to build it just once, and *copy* (or make a symlink) it to the `build/<mode>` directories.